other/parser_build.yml

```diff
@@ -0,0 +1,35 @@
+trigger: none
```

Comment:
**Misleading name**: Also, don't like the name "parser" here. Tilck is medium-sized project with a ton of stuff inside. Parser doesn't mean anything. The file name should be something like `cmake_configurator.yml`